Source/Extensions/Blazorise.DataGrid/_DataGridCellEdit.razor.cs

```diff
@@ -6,20 +6,47 @@
 
 namespace Blazorise.DataGrid
 {
+    /// <summary>
+    /// Internal component for editing the row item cell value.
+    /// </summary>
+    /// <typeparam name="TItem"></typeparam>
     public partial class _DataGridCellEdit<TItem> : ComponentBase
     {
+        /// <summary>
+        /// Updated the internal cell values.
+        /// </summary>
+        /// <typeparam name="T">Type of the value being changed.</typeparam>
+        /// <param name="value">Value that is updating.</param>
+        /// <returns>A task that represents the asynchronous operation.</returns>
         protected Task OnEditValueChanged<T>( T value )
         {
             CellEditContext.CellValue = value;
 
+            Column.SetValue( ValidationItem, value );
+
             return CellValueChanged.InvokeAsync( value );
         }
 
         /// <summary>
-        /// Row model.
+        /// Column that this cell belongs to.
+        /// </summary>
+        [Parameter] public DataGridColumn<TItem> Column { get; set; }
+
+        /// <summary>
+        /// Field name that this cell belongs to.
+        /// </summary>
+        [Parameter] public string Field { get; set; }
+
+        /// <summary>
+        /// Instance of the currently editing row item.
         /// </summary>
         [Parameter] public TItem Item { get; set; }
 
+        /// <summary>
+        /// Instance of the currently validating row item.
+        /// </summary>
+        [Parameter] public TItem ValidationItem { get; set; }
```

Comment:
Potentially just grab out of Cascading Datagrid? I generally prefer using the Cascading parameter for internal values, over having to set the parameters manually all the way down. What do you think?

Reply:
Wanted to follow the same pattern as we already have, but I agree it makes more sense to cascade. I will fix it.

Reply:
In fact, better to leave it as a parameter. There is a feature request for editing multiple rows at once and with parameters it would be a lot easier to handle it once we start working on it.